config/development.json

```
@@ -10,6 +10,5 @@
 	"SYNC_LOG_LEVEL": "debug",
 	"REQUEST_LOGGING_ENABLED": true,
 	"FEATURE_ALERTS_STATUS_ENABLED": true,
-	"FEATURE_MONGOOSE_LOGGING_ENABLED": true,
-	"LERNSTORE_MODE": "DISABLED"
+	"FEATURE_MONGOOSE_LOGGING_ENABLED": true
```

Before, the Lernstore was disabled on development by default.
```suggestion
	"FEATURE_MONGOOSE_LOGGING_ENABLED": true,
	"FEATURE_LERNSTORE_ENABLED": false,
```

---

fixed